docs/CHANGELOG.md

```diff
@@ -30,6 +30,10 @@ nav_order: 5
 
     *Anton Prins*
 
+* Add support for sub-templates.
```

Comment:
```suggestion
* Add experimental support for sub-templates.
```

Reply:
Is the plan to make this an opt-in feature like we have with other experimental features?

I'm still hesitant about the tradeoffs being worth introducing yet another rendering concept to VC, especially since slots aren't as well understood as they could be. Happy to be overruled, but if we do plan on adding this feature I'd love to see it as an opt-in experiment like we've done with other features before committing to it.